<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextForm</class>
 <widget class="QWidget" name="TextForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#TextForm{
	

	background-color: rgb(57, 56, 61);
}
QFrame{
	border:none;
}
QFrame#frameText{
	
	background-color: rgb(68, 68, 68);
}
QFrame#frameFont{
	
	
	background-color: rgb(108, 108, 108);
	min-height:36px;
}
QFrame#frameBtn{
	
	background-color: rgb(117, 117, 117);
	min-height:28px;
}

QPlainTextEdit{
	background-color: rgb(230, 230, 230);
}
QFrame#frameBtn QPushButton{
	min-height:28px;
	min-width:60px;
	border:none;
	
	font: 9pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(98, 98, 98);
}
QToolButton{
	border:none;
	background-color: rgb(108, 108, 108);
	
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;微软雅黑&quot;;
}
QToolButton#btnItly{
	border:none;
	background-color: rgb(108, 108, 108);
	
	color: rgb(255, 255, 255);
	
	font: italic 12pt &quot;微软雅黑&quot;;
}
QComboBox#comboBoxFontSize{
	
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QFrame" name="frameText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>417</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameFont">
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtBottom</enum>
         </property>
         <property name="minimumContentsLength">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnBlod">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icon.qrc">
           <normaloff>:/Source/icon/text_b.png</normaloff>:/Source/icon/text_b.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFontSize">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>20</number>
         </property>
         <property name="maxVisibleItems">
          <number>10</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>36</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>42</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>56</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>72</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>96</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>180</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>39</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <widget class="QFrame" name="frameBtn">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>274</width>
           <height>19</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOk">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
